integreat_cms/cms/templates/events/event_list.html

```diff
@@ -130,6 +130,9 @@ <h1 class="heading">{% trans 'Events' %}</h1>
                         </option>
                     {% endif %}
                 {% endif %}
+                <option data-bulk-action="{% url 'simple_german_translation_pages' %}" title=" {% trans 'Automatically translate this page to easy language' %}">
+                    {% trans 'Automatically translate this page to easy language' %}
```

Comment:
```suggestion
                    {% trans 'Automatically translate this page to simple German' %}
```
The API only handles German and no other languages, right?